sormas-api/src/main/java/de/symeda/sormas/api/campaign/diagram/CampaignDiagramSeries.java

```diff
@@ -10,21 +10,28 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 import de.symeda.sormas.api.AgeGroup;
+import de.symeda.sormas.api.audit.AuditInclude;
+import de.symeda.sormas.api.audit.AuditedClass;
 import de.symeda.sormas.api.i18n.Validations;
 
+@AuditedClass
```

Comment:
Don't audit this class or at least not those fields

Reply:
only class audited now